src/librustc_error_codes/error_codes/E0639.md

```diff
@@ -3,5 +3,17 @@ instantiated from outside of the defining crate as it has been marked
 as `non_exhaustive` and as such more fields/variants may be added in
 future that could cause adverse side effects for this code.
 
+Erroneous code example:
+
+```ignore (not yet an error apparently)
```

Comment:
This only works cross crate, not sure how to do that in a doc test

Reply:
Within the same crate, exhaustiveness is not a problem, that's why it's not erroring

Reply:
We can just keep it ignored, but I should update the message.